app/javascript/packs/clipboardCopy.js

```diff
@@ -1 +1,10 @@
-import 'clipboard-copy-element';
+document.addEventListener('DOMContentLoaded', () => {
+  const waitingForOnboarding = setInterval(() => {
```

Comment:
question: why did you wrap it in a set interval? doesn't it already "wait" ? I'm looking at the example you mentioned here https://github.com/webcomponents/webcomponentsjs/#asynchronous

Reply:
Thanks for pointing that out. I've tried so many things but I forgot to try that 😆. I updated it and it seems to be working too.

Reply:
I now remember why, Waterfox doesn't seem to acknowledge the waitfor method.